src/encoded/static/components/browse/BrowseView.js

```diff
@@ -494,33 +316,26 @@ function BrowseTableWithSelectedFilesCheckboxes(props){
         windowHeight, windowWidth, registerWindowOnScrollHandler,
         toggleFullScreen, isFullscreen, session,
 
-        // Props from BrowseView:
-        facets,
-
         // Props from SelectedFilesController (separate from SPC SearchView SelectedItemsController):
-        selectedFiles, selectFile, unselectFile, resetSelectedFiles,
+        selectedFiles, selectFile, unselectFile, resetSelectedFiles, selectedFilesUniqueCount,
```

Comment:
Codacy found an issue: ['resetSelectedFiles' is missing in props validation](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6342004)